pipeline/config/exoscale/sc-config.yaml

```diff
@@ -184,6 +184,16 @@ prometheus:
     tolerations: []
     affinity: {}
     nodeSelector: {}
+  grafana:
+    resources:
+      requests:
+        cpu: 100m
+        memory: 128Mi
+      limits:
+        cpu: 100m
+        memory: 128Mi
+    oidc:
+      enabled: false
```

Comment:
Have you tested that the chart will install ok without specifying the other oidc values? I'm thinking that it might complain that some values are missing.

Reply:
You are correct, that didn't work. I'll add some defaults for those as well